Add explanations on how to fix `package.json` for `puppeteer-core`

romaricpascal opened the pull request:
Following on https://github.com/alphagov/govuk-frontend/pull/5750, figuring out what to update when the test fails was a little tricky, so I've added a comment and updated the test title to hopefully clarify.

domoscargin commented on package-lock.json.unit.test.mjs:
```diff
@@ -55,7 +55,16 @@ describe('package-lock.json', () => {
+     * with that of `puppeteer` in the other workspaces (usually the version that Dependabot
+     * will have bumped `puppeteer` to).
+     *
+     * {@link https://team-playbook.design-system.service.gov.uk/how-we-work/version-control/pull-requests#reviewing-a-pr-from-dependabot}
```
Broken link for me

romaricpascal replied:
Oh, indeed, it has moved to https://team-playbook.design-system.service.gov.uk/guides/version-control/pull-requests/#reviewing-a-pr-from-dependabot. I'll update :)

domoscargin commented on package-lock.json.unit.test.mjs:
```diff
@@ -55,7 +55,16 @@ describe('package-lock.json', () => {
+     * by checking out the Dependabot branch and updating `shared/helpers/package.json`
+     * so that the version of `puppeteer-core` in `peerDependencies` is compatible
+     * with that of `puppeteer` in the other workspaces (usually the version that Dependabot
+     * will have bumped `puppeteer` to).
```
Should we explicitly mention running `npm install` again, which I'm guessing you have to do?

romaricpascal commented:
@domoscargin I've updated the link and added the instruction to run `npm install`. I've also aligned the format of the comment for the test a little further up (and fixed the link there as well).

domoscargin reviewed (approved):
👨‍🍳
